packages/app/src/app/overmind/namespaces/dashboard/actions.ts

```diff
@@ -1928,91 +1931,69 @@ export const getContributionBranches = async ({ state, effects }: Context) => {
 
 type RepositoriesActionOptions = {
   teamId: string;
-  bypassLoading?: boolean;
+  syncData?: boolean;
```

Comment:
decided to pass directly the `syncData` flag as a param and call this function twice if needed, instead of duplicating the logic inside it

Reply:
Like the name change!